src/components/timeselector/TimeSelectorDirective.js

```diff
@@ -247,11 +247,19 @@ goog.require('ga_time_service');
               var year = scope.options.years[i];
               year.available = false;
               olLayers.forEach(function(olLayer, opt) {
-                if (year.available || !olLayer.bodId) {
+                if (year.available ||
+                    (!olLayer.bodId &&
+                     !gaMapUtils.isExternalWmtsLayer(olLayer.id))) {
                   return;
                 }
-                var timestamps = gaLayers.getLayerProperty(olLayer.bodId,
-                    'timestamps') || [];
+                var timestamps;
+                if (gaMapUtils.isExternalWmtsLayer(olLayer.id)) {
+                  timestamps = olLayer.timestamps;
+                } else {
+                  timestamps = gaLayers.getLayerProperty(olLayer.bodId,
+                    'timestamps');
+                }
+                timestamps = timestamps || [];
```

Comment:
No need of the `if`and `else`use the  timestamps property of olLayer .  `var timestamps = olLayer.timestamps || []`;